src/js/containers/covid19/recipient/SpendingByRecipientContainer.jsx

```diff
@@ -183,9 +183,9 @@ const SpendingByRecipientContainer = ({ activeTab }) => {
         if (activeTab !== 'all') {
             params.filter.award_type_codes = awardTypeGroups[activeTab];
         }
-        let request = fetchSpendingByRecipient(params);
+        let request = fetchDisasterSpending('recipient', params);
         if (activeTab === 'loans') {
-            request = fetchRecipientLoans(params);
+            request = fetchLoanSpending('recipient', params);
```

Comment:
It looks like we're fetching this request twice in the browser. Could we implement the cancel pattern here?